packages/paste-core/components/tooltip/stories/index.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import {useUID} from 'react-uid';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Absolute} from '@twilio-paste/absolute';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Text} from '@twilio-paste/text';
+import {Tooltip} from '../src';
+
+const Example: React.FC<{}> = () => {
+  const id = useUID();
+  return (
+    <Tooltip baseId={id} text="Welcome to Paste!">
+      <Button variant="primary">Open Tooltip</Button>
+    </Tooltip>
+  );
+};
+
+const IconButtonExample: React.FC<{}> = () => {
+  const id = useUID();
+  return (
+    <Tooltip baseId={id} text="Welcome to Paste!">
+      <Button variant="secondary" size="icon">
+        <InformationIcon decorative={false} title="Open Tooltip" />
```

Comment:
This is an interesting example... What happens if you hover the icon svg? Does the title appear behind the tooltip?

Would it be appropriate here to match the icon title to the tooltip text?

Reply:
It looks like the title appears above the tooltip, even with the highest zIndex. Maybe we add some guidance in the doc page about matching the title to the tooltip text. What do you think @TheSisb?

Reply:
I _think_ we can mark the icon as decorative and leverage the tooltip to inform users, but I haven't tested that theory.

Documentation would be great yeah 👍

Reply:
@TheSisb I adjusted the doc and story some to inform users more about leveraging an icon.

Reply:
> I think we can mark the icon as decorative and leverage the tooltip to inform users

Unfortunately, _technically_ not. A tooltip should describe the thing you are pointing at, but if you remove the text content of the thing you’re describing, there’s nothing to describe. Making it worse is decorative icons are essentially removed from the DOM to screen reader users.

Reply:
@SiTaggart should the icon title match the tooltip text? Or be more like "open tooltip"?

Reply:
@SiTaggart Isn't it safe in the fact that the icon is removed from the DOM? The only thing to point to is the button, which will have the correct tooltip information.

Reply:
So the nuance is the accessible name the icon is giving the button element. If you set the icon to be decorative we `aria-hidden="true"` the icon, which removes the accessible name from the button. Effectively it removes the button text entirely, so the tooltip _describes_ a labelless button.

Often the tooltip text should not be the same as the button text, if you are describing it. It might be a pencil icon for an edit button, the button text might be "Edit" and the tooltip might describe what that means. They are independent pieces of information. 

![image](https://user-images.githubusercontent.com/368249/86191630-622f1880-bafc-11ea-949f-2f6ae06bdca0.png)

![image](https://user-images.githubusercontent.com/368249/86192010-5c860280-bafd-11ea-9fce-828f545c9b7e.png)

Reply:
Have found a bug though, so that's exciting. As you can see in the screen shots the Description is "Open Tooltip" which is incorrect, it should be "Welcome to Paste". 

Reason is duplicate ids. The icon id's are unique but independently unique to the tooltip ids generated by reakit. I think we just need to set a default baseId to prefix `paste-tooltip` or something to the tooltip ids

![image](https://user-images.githubusercontent.com/368249/86192428-73792480-bafe-11ea-92ad-a642481b1db9.png)

Reply:
Makes sense and good catch! We should prefix the icon IDs to `paste-icon-#` too 🤔

Reply:
@SiTaggart I've added a default baseId prefixed with `paste-tooltip-`, and using `useUID` to generate the number.